SConstruct

```diff
@@ -1247,6 +1247,23 @@ def inkscapeArgs( imageOptions, svg ) :
 	width = int( round( svgObjectInfo["width"] ) )
 	height = int( round( svgObjectInfo["height"] ) )
 
+	# Ensure images are properly pixel aligned and optionally, a specific size.
+	# Transparent container objects should be used where the artwork is of a shape that precludes this.
+
+	if imageOptions.get( "validatePixelAlignment", True ):
+		if width != svgObjectInfo["width"] or height != svgObjectInfo["height"] :
+			raise RuntimeError(
+				"Object with id '%s' is not aligned to pixel boundaries w: %s h: %s" %
+					( id_, svgObjectInfo["width"], svgObjectInfo["height"] )
+			)
+
+	# Optional exact dimension validation
+
+	vw = imageOptions.get( "validateWidth", None )
```

Comment:
`validateWidth` sounds like a bool, in the same way that `validatePixelAlignment` is. Perhaps `requiredWidth`?

Reply:
Fixed [here](https://github.com/GafferHQ/gaffer/pull/3874/commits/d2a996ee2bc2d6e35b897965ec80ce3e8b994064)